IntelliTect.TestTools.TestFramework/IntelliTect.TestTools.TestFramework.Tests/TestData/Dependencies/ExampleInterface.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace IntelliTect.TestTools.TestFramework.Tests.TestData.Dependencies
+{
+    public interface IExampleDataInterface
+    {
+        string Testing { get; set; }
+    }
+
+    public class ExampleInterface : IExampleDataInterface
+    {
+        public string Testing { get; set; } = "Testing";
+    }
+
+    public class OtherExampleInterface : IExampleDataInterface
```

Comment:
nit: These both have "Interface" in their name but are classes. Is there a better name that could be used here?